### PR TITLE
on changing symbol, assert that comparison series are removed from Ch…

### PR DESCRIPTION
**Pull request by brucebinary**
…artIQ before creating new chart

### CODE REVIEW

**Comment by brucebinary** [comment]
@MahboobehMohammadi you can close this PR when you're done with it. It's never intended to be a final solution

File: src/store/ChartStore.js [hunk 1]
```diff
@@ -419,6 +419,14 @@ class ChartStore {
             params = { periodicity: calculateTimeUnitInterval(granularity) };
         }
 
+        if (params === undefined && symbolObj) {
+            for (const field in this.stxx.chart.series) {
+                if (this.stxx.chart.series[field].parameters.bucket !== 'study') {
```
Comment (MahboobehMohammadi) [inline src/store/ChartStore.js]: @brucebinary It looks fine and works fine, thanks! except I don't think chartiq add series for indicators

Comment (brucebinary) [reply src/store/ChartStore.js]: @MahboobehMohammadi this line is from their original source code some versions back. It could be outdated though.

**Comment by morteza-binary** [comment]
Duplicate of #650

**Pull request closed without merging**